include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -1,10 +1,10 @@
 #ifndef STATSD_CLIENT_HPP
 #define STATSD_CLIENT_HPP
 
+#include <cpp-statsd-client/UDPSender.hpp>
```

Comment:
matching the installation which has the subdirectory and using the `<>` brackets to search the system include paths first